smrf/data/modifiers.py

```diff
@@ -0,0 +1,43 @@
+def get_scalar_value_key(config, stage):
+    """
+    Get the config key that holds the values defining a data scalar approach
+    Args:
+        config: the variable specific config section
+        stage: the stage that the scaling is being applied to. Allowed values
+            are ["input", "output"]
+
+    Returns:
+        scalar_value_key: the config key that can access the values needed to
+            scale the data
+    """
+    if stage not in ["input", "output"]:
+        raise ValueError(
+            f"{stage} is not an allowed stage value for data scaling"
+        )
+    method = config.get(f"{stage}_scalar_type")
+    if method is None:
+        return None
+    return f"{stage}_scalar_{method}"
+
+
+def scale_data(data, scalar_key, config):
```

Comment:
This can definitely be simplified. I wanted to leave room for more scaling methods down the line, but I can definitely avoid passing the config here. Making the `Modifiers` class will also help with this.